src/wrapper.rs

```diff
@@ -547,6 +547,8 @@ fn is_message(attrs: &[Attribute]) -> bool {
             let tts = a.tts.to_string();
             if tts.contains(":: prost_derive :: Message") {
                 return true;
+            } else if tts.contains(":: prost :: Message") {
+                return true;
```

Comment:
I think just using `tts.contains("Message")` is a better solution, there's little point in being totally strict here.

Reply:
Fixed